keras_cv/layers/preprocessing/cut_mix_test.py

```diff
@@ -79,6 +100,89 @@ def test_cut_mix_call_results_one_channel(self):
         self.assertNotAllClose(ys, 1.0)
         self.assertNotAllClose(ys, 0.0)
 
+    def test_cut_mix_call_results_with_masks(self):
+        xs = tf.cast(
+            tf.stack(
+                [2 * tf.ones((4, 4, 3)), tf.ones((4, 4, 3))],
+                axis=0,
+            ),
+            tf.float32,
+        )
+
+        ys_segmentation_masks = tf.cast(
+            tf.stack(
+                [2 * tf.ones((4, 4)), tf.ones((4, 4))],
+                axis=0,
+            ),
+            tf.uint8,
+        )
+        ys_segmentation_masks = tf.one_hot(ys_segmentation_masks, 3)
+
+        layer = CutMix(seed=1, apply_to_segmentation_masks=True)
+        outputs = layer(
+            {"images": xs, "segmentation_masks": ys_segmentation_masks}
+        )
+        xs, ys_segmentation_masks = (
+            outputs["images"],
+            outputs["segmentation_masks"],
+        )
+
+        # At least some pixels should be replaced in the images
+        self.assertTrue(tf.math.reduce_any(xs[0] == 1.0))
+        self.assertTrue(tf.math.reduce_any(xs[0] == 2.0))
+        self.assertTrue(tf.math.reduce_any(xs[1] == 1.0))
+        self.assertTrue(tf.math.reduce_any(xs[1] == 2.0))
+        # At least some pixels should be replaced in the segmentation_masks
+        self.assertTrue(tf.math.reduce_any(ys_segmentation_masks[0] == 1.0))
+        self.assertTrue(tf.math.reduce_any(ys_segmentation_masks[0] == 0.0))
+        self.assertTrue(tf.math.reduce_any(ys_segmentation_masks[1] == 1.0))
+        self.assertTrue(tf.math.reduce_any(ys_segmentation_masks[1] == 0.0))
+
+    def test_cut_mix_call_results_one_channel_with_masks(self):
```

Comment:
This test case doesn't seem to add a lot of coverage, so we can probably drop it.